### DISCUSSION TITLE
Stuck on rebuilding in terminal?

### DISCUSSION THREAD

**Discussion by terrencejay** [tailwindlabs/tailwindcss#13933] [Help]
the command I am running is npx tailwindcss -i ./src/input.css -o ./src/output.css --watch
im using mac react version 18.3.1.

the terminal is stuck in rebuilding and I am not seeing any of the changes I am applying to my code. I.E '<h1 className="text-3xl font bold .bg-red-500">
hello world
</h1>'
is this supposed to happen? is there something wrong with my code?

**Comment by kugua174** [comment]
me too

**Comment by kugua174** [comment]
plz tell me when u find the solution
